src/main/java/gift/product/controller/AdminController.java

```diff
@@ -1,65 +1,128 @@
+import gift.product.dto.MemberDto;
 import gift.product.model.Product;
+import gift.product.service.AuthService;
 import gift.product.service.ProductService;
+import jakarta.servlet.http.Cookie;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
 import jakarta.validation.Valid;
+import java.net.URI;
 import java.util.List;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
 
 @Controller
-@RequestMapping("/admin/products")
+@RequestMapping("/admin")
 public class AdminController {
 
-    public static final String REDIRECT_ADMIN_PRODUCTS = "redirect:/admin/products";
+    private final String REDIRECT_ADMIN_LOGIN_PROCESSING = "redirect:/admin/login/process";
+    private final String REDIRECT_ADMIN_PRODUCTS = "redirect:/admin/products";
     private final ProductService productService;
+    private final AuthService authService;
 
-    public AdminController(ProductService productService) {
+    public AdminController(ProductService productService, AuthService authService) {
         this.productService = productService;
+        this.authService = authService;
     }
 
-    @GetMapping
-    public String products(Model model) {
-        List<Product> products = productService.getProductAll();
+    @GetMapping("/login")
+    public String loginForm() {
+        return "admin/loginForm";
+    }
+
+    @PostMapping("/login")
+    public String login(MemberDto memberDto, RedirectAttributes redirectAttributes) {
+        JwtResponse jwtResponse = authService.login(memberDto);
+        redirectAttributes.addAttribute("accessToken", jwtResponse.token());
+
+        return REDIRECT_ADMIN_LOGIN_PROCESSING;
+    }
+
+    @GetMapping("/login/process")
+    public String loginProcess(@RequestParam("accessToken") String accessToken, Model model) {
+        model.addAttribute("accessToken", accessToken);
+
+        return "admin/loginProcess";
+    }
+
+    @PostMapping("login/process")
+    @ResponseBody
+    public ResponseEntity<?> loginSuccess(@RequestParam("accessToken") JwtResponse jwtResponse, HttpServletResponse response) {
```

Comment:
와일드카드 타입을 넣는 것보단 Void를 넣는 방법도 있습니다~!
```suggestion
    public ResponseEntity<Void> loginSuccess(@RequestParam("accessToken") JwtResponse jwtResponse, HttpServletResponse response) {
```

Reply:
조언 감사합니다! 반영하겠습니다.